components/MUI/getMUITheme.ts

```diff
@@ -12,18 +12,49 @@ export type ThemeMode = Theme["palette"]["mode"];
 const spacing = 4;
 // TODO: I'm not sure whether it should be 4 or 5
 
+const primaryColors = {
+  orange: "#fe6C0a",
+  darkGrey: "#3a3a3a",
+  lightGrey: "#f2f5f7",
+  white: "#ffffff",
+  darkBlack: "#18191d",
+  lightBlack: "#33393f",
+  blue: "#3c93bf",
+  black: "#000000",
+};
+
 const lightThemeConfig = createTheme({
   palette: {
     mode: "light",
-    // TODO: We need to add colors from Figma
+    primary: {
```

Comment:
@1alexvash We can't do this because primary has the unique interface 
<img width="416" alt="Снимок экрана 2023-03-29 в 21 32 46" src="https://user-images.githubusercontent.com/79473861/228635093-77662993-0541-4b32-99e1-208550091feb.png">

Reply:
Or something I don't understand, is it?